### PR TITLE
Add float_vector psycopg3 test

### PR DESCRIPTION
**Pull request by matriv**

### CODE REVIEW

**Comment by matriv** [comment]
`Python client tests` succeed, the failures are related to: https://github.com/crate/crate/commit/1cb03b79e0c5b4d49b24ec8fefaa45642bf3db33

File: tests/client_tests/python/asyncpg/test_asyncpg.py [hunk 1]
```diff
@@ -80,6 +80,18 @@ async def bitstring_can_be_inserted_and_selected_using_binary_encoding(test, con
...
+async def float_vector_can_be_inserted_and_selected(test, conn):
+    xs = [1.1, 2.2, 3.3, 4.4]
+    await conn.execute('drop table if exists tbl_fv')
+    await conn.execute('create table tbl_fv (id int, xs float(4))')
+    await conn.execute('insert into tbl_fv (xs) values (1, ?)', xs)
+    await conn.execute('insert into tbl_fv (xs) values (2, ?)', None)
+    await conn.execute('refresh table tbl_fv')
+    result = await conn.fetch('select xs from tbl_fv order by id')
+    test.assertEqual(result[0][0], xs)
+    test.assertNone(result[0][0])
```
Comment (BaurzhanSakhariev) [inline tests/client_tests/python/asyncpg/test_asyncpg.py:92]: Shouldn't it be `test.assertNone(result[1][0])`?

Comment (matriv) [reply tests/client_tests/python/asyncpg/test_asyncpg.py:92]: yes, thank you! I have to check

**Comment by matriv** [comment]
Opened issue to track this: https://github.com/crate/crate-python/issues/581

**Review by BaurzhanSakhariev — APPROVED** [review]
👍

**Pull request merged**